Sources/TuistKit/Commands/CommandRegistry.swift

```diff
@@ -75,7 +76,10 @@ public final class CommandRegistry {
 
                 // Normal command
             } else {
-                let parsedArguments = try parse()
+                guard let parsedArguments = try parse() else {
+                    parser.printUsage(on: stdoutStream)
```

Comment:
This is a good idea. We should do the same with other commands. Not as part of this PR though.

Reply:
This change is necessitated but this PR. `parse()` now returns optional `ArgumentParser.Result` if this does not include a command name. I can probably change this implementation back to its original signature.